### ISSUE TITLE
add dtb support x96max+ s905x3_4g_1gbit

### ISSUE THREAD

**Issue by navigos** [ophub/amlogic-s9xxx-armbian#758]
**Device Information | 设备信息**
- x96-max-plus / sm1_s905x3_4g_1gbit
- #247 

**Armbian Version | 系统版本**
 - Armbian_Aml_jammy_x
 - Armbian_Aml_bullseye_x

**Describe the bug | 问题描述**
- Current versions (jammy and bullseye / kernel 5.10.156 and 5.15.80) do not support the attached dtb file.
- [sm1_s905x3_4g_1gbit.zip](https://github.com/ophub/amlogic-s9xxx-armbian/files/10144708/sm1_s905x3_4g_1gbit.zip)
- I can run EmuELEC with this dtb file
- The armbian_ophub kernel, 5.4 worked fine.

Thank for your help!

**Comment by ophub** [comment]
This driver has been added in several kernels. If you test that 5.10 cannot be used, use 5.4.

**Comment by navigos** [comment]
Perhaps I am not applying the procedure incorrectly.

- I tried with standard files;
- I tried copying and renaming the u-boot-* files to u-boot.ext;
- I tried changing line 3 of the uEnv.txt file (meson-sm1-x96-max-plus-100m.dtb/meson-sm1-x96-max-plus.dtb/meson-sm1-x96-air-gbit.dtb/meson -sm1-x96-air.dtb/meson-sm1-sei610.dtb);
- I tried changing line 1 of the uEnv.txt file (uImage/zImage/Image).

Are any of these procedures wrong?

ps: Is it possible to get Armbian with kernel 5.4? I looked in Releases but I didn't find it and I don't have the knowledge to compile one.
